src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportProcessingJob.cs

```diff
@@ -167,6 +178,18 @@ public async Task<string> ExecuteAsync(JobInfo jobInfo, CancellationToken cancel
                 var error = new ImportJobErrorResult() { ErrorMessage = ex.Message, HttpStatusCode = HttpStatusCode.BadRequest, ErrorDetails = ex.ToString() };
                 throw new JobExecutionException(ex.Message, error, ex);
             }
+            catch (IntegrationDataStoreException ex) when (ex.Message.Contains("The specified resource name contains invalid characters", StringComparison.OrdinalIgnoreCase))
```

Comment:
Do we need a length check on the container name? Limit is 1024 chars or 256 for azure emulator. Edge case for sure but possible.

Reply:
There is one on line 187. The limits are actually much smaller, between 3 and 63 characters.